core/src/main/scala/org/graphframes/lib/AggregateMessages.scala

```diff
@@ -59,9 +60,13 @@ import org.graphframes.Logging
  */
 class AggregateMessages private[graphframes] (private val g: GraphFrame)
     extends Arguments
-    with Serializable {
+    with Serializable
+    with WithIntermediateStorageLevel
+    with Logging {
 
-  import org.graphframes.GraphFrame.{DST, ID, SRC}
```

Comment:
Just curious why this was made into three imports? Strange change.

Reply:
It is how scalafmt is configured :)